src/controller.js

```diff
@@ -29,7 +49,6 @@ const getPrint = async (req, res, next) => {
     if (!id || !ObjectId.isValid(id)) {
       return res.status(422).send('error: invalid space id');
     }
-
     const { query, headers } = req;
```

Comment:
Since we are doing a `post` now, we should actually pass `format` and `id` inside the body and not as a URL param or a query param.